dangerfile.js

```diff
@@ -39,53 +39,50 @@ const tasks = [
     function unitTests() {
         try {
             execa.sync('jest', ['--json', '--coverage']);
-            const coverageLink = linkToCircleAsset(
-                'coverage/lcov-report/index.html'
-            );
-            markdown(
-                `All tests passed! [View Coverage Report](${coverageLink})`
-            );
         } catch (err) {
             const summary = JSON.parse(err.stdout);
             const failedTests = summary.testResults.filter(
                 t => t.status !== 'passed'
             );
             // prettier-ignore
             const failSummary = failedTests.map(t =>
-`<details>
-<summary>${fromRoot(t.name)}</summary>
-<pre>${t.message}</pre>
-</details>`
-            ).join('\n');
+    `<details>
+    <summary>${fromRoot(t.name)}</summary>
+    <pre>${t.message}</pre>
+    </details>`
+                ).join('\n');
             fail(
                 'The following unit tests did _not_ pass 😔. ' +
                     'All tests must pass before this PR can be merged\n\n\n' +
                     failSummary
             );
         }
-    },
-
-    function storybook() {
-        const storybookURI = linkToCircleAsset('storybook-dist/index.html');
-        markdown(
-            `[A Storybook for this PR has been deployed!](${storybookURI}). ` +
-                'It will be accessible as soon as the current build completes.'
-        );
     }
-];
 
-for (const task of tasks) task();
+    // Disabled for now, but leaving in for future implementation.
+    // Can't use right now due to the lack of permissions granularity
+    // in GitHub
+    // async function addProjectLabels() {
+    //     const allChangedFiles = [
+    //         ...danger.git.created_files,
```

Comment:
Hmm, what's the permissions granularity issue?

Reply:
To add labels you have to be an admin with write access. Any API key we expose in Circle could be accessed by someone opening a PR, so we can't give `PWAStudioBot` those permissions